src/mdFormBuilder.js

```diff
@@ -145,7 +145,9 @@ function FormBuilderCtrl ($scope, $window, $timeout, $anchorScroll, $location, h
     for (var k in form) {
       if (form.hasOwnProperty(k)) {
         if (typeof form[k] === 'object' && form[k].hasOwnProperty('$modelValue')) {
-          formFieldsValues[k] = form[k].$modelValue
+          if (form[k].$modelValue !== '' && form[k].$modelValue !== undefined) {
```

Comment:
@rcrichton is there a reason you didn't just say if (form[k].$modalValue)?  Just wondering about the null case too.

Reply:
Yeah, because if `form[k].$modelValue` is a legitimate value of `false` then it would be skipped.

I can add a null check to.